eo-maven-plugin/src/test/java/org/eolang/maven/UnphiMojoTest.java

```diff
@@ -335,6 +329,11 @@ void invalidatesCache(@Mktmp final Path temp) throws Exception {
             "{⟦std ↦ Φ.org.eolang.io.stdout, y ↦ Φ.org.eolang.x⟧}",
             temp.resolve("target/eo/phi/std.phi")
         ).value();
+        MatcherAssert.assertThat(
+            "The cached file's last modified timestamp should be successfully reset",
+            cached.setLastModified(0L),
```

Comment:
@vl4ds4m what's the reason to change this timestamp? Hard coded cached file was saved before the the original `std.phi`. So it'll be anyway younger